dependencies/parsedown-extra/ParsedownExtra.php

```diff
@@ -573,6 +573,17 @@ protected function processTag($elementMarkup) # recursive
         # http://stackoverflow.com/q/11309194/200145
         $elementMarkup = htmlentities($elementMarkup);
 		$elementMarkup = htmlspecialchars_decode($elementMarkup);
```

Comment:
Shouldn't these two lines be removed then?

Reply:
I thought the snippet you gave is only for emoji. It replaces those two lines too?

Reply:
If I read the StackOverflow thread linked in the comment above, it only points to the fact that `DOMDocument` does not support UTF-8 characters, which is why everything non-ASCII was converted to an entity. This is what my new code does (converting everything with 8 bits or more to an entity, leaving ASCII alone).